runtime/doc/lsp.txt

```diff
@@ -1197,32 +1197,33 @@ format({options})                                       *vim.lsp.buf.format()*
                                • formatting_options (table|nil): Can be used
                                  to specify FormattingOptions. Some
                                  unspecified options will be automatically
-                                 derived from the current Neovim options.
-
-                See also: ~
-                    https://microsoft.github.io/language-server-protocol/specification#textDocument_formatting
-                    • timeout_ms (integer|nil, default 1000): Time in
-                      milliseconds to block for formatting requests. No effect
-                      if async=true
-                    • bufnr (number|nil): Restrict formatting to the clients
-                      attached to the given buffer, defaults to the current
-                      buffer (0).
-                    • filter (function|nil): Predicate used to filter clients.
-                      Receives a client as argument and must return a boolean.
-                      Clients matching the predicate are included. Example:    • >
+                                 derived from the current Neovim options. See https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#formattingOptions
```

Comment:
Is there some clever trick to get the docs generator place the link on the next line?

Reply:
no